src/pages/Main/components/SortBar/SortBar.js

```diff
@@ -0,0 +1,69 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+const SortBar = ({ handleSort }) => {
+  const [click, setClick] = useState('');
+
+  const sortBtn = [
+    {
+      id: 1,
+      name: '신상품순',
```

Comment:
```suggestion
  const SORT_BTNS = [
    {
      id: 1,
      name: '신상품순',
```
상수를 선언할 때는 거기에 맞는 컨벤션을 지켜주셔야 합니다.
상수 컨벤션은 대문자 + snake_case 입니다.
또한 가독성을 위해 상수를 선언하는 부분을 문서의 최하단으로 옮겨주세요!